tests/dags/providers/provider_api_scripts/resources/JsonLoad.py

```diff
@@ -0,0 +1,8 @@
+import json
+from pathlib import Path
+
+
+def get_resource_json(folder_name, json_name):
+
+    RESOURCES = Path(__file__).parent / folder_name
+    return json.loads((RESOURCES / json_name).read_text())
```

Comment:
While the `RESOURCES` variable was caps-case for many of the now-removed functions, we typically reserve usage of caps-case to global variables. This can be lower case 🙂 
```suggestion
    resources = Path(__file__).parent / folder_name
    return json.loads((resources / json_name).read_text())
```